### PR TITLE
inital commit of FiVL

### PR DESCRIPTION
**Pull request by michaelbeale-IL**

### CODE REVIEW

File: FiVL/pipeline/augment_dataset/run_main.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#SBATCH --qos=high
+#SBATCH --job-name=rm_665k_gpt
+#BATCH -w isl-gpu44
+#SBATCH -x isl-gpu42,isl-gpu26,isl-gpu27
+#SBATCH --mail-type=ALL  #BEGIN, END, FAIL,ALL
+#SBATCH --mail-user=gabriela.ben.melech.stan@intel.com
+#SBATCH --output=slurm-%x.%j.out
+
+# input the data json file
+JSON_FILE="/home/gbenmele/lvlm/testfiles/splits/test_data.json"
```
Comment (estelleafl) [inline FiVL/pipeline/augment_dataset/run_main.sh]: @gbenms  gabi coud you make all the ref to your workspace to something that anyone can use?

File: FiVL/pipeline/augment_dataset/slurm-153829.out
Comment (estelleafl) [inline FiVL/pipeline/augment_dataset/slurm-153829.out:1]: can we remove this? @gbenms

File: FiVL/pipeline/augment_dataset/utils_preprocess_image.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    device='cuda'
+    image_processor = CLIPProcessor.from_pretrained(pretrained_model_name_or_path=model_path, device=device)
+    image = Image.open(image_path).convert("RGB") 
+    images = [image]
+    images = process_images(images, image_processor.image_processor, image_aspect_ratio)
+    pil_image = to_pil_image(images[0])
+
+    return pil_image
+
+# def convert_mask_image_to_mask_vector(mask_image_path, num_patches_y = 24, num_patches_x = 24):
```
Comment (estelleafl) [inline FiVL/pipeline/augment_dataset/utils_preprocess_image.py]: can you remove this?

File: FiVL/pipeline/augment_dataset/run_main.sh (hunk 1 above)
Comment (estelleafl) [reply FiVL/pipeline/augment_dataset/run_main.sh]: or even better, can you please maybe turn this into a README?

**Review by estelleafl — APPROVED** [review]

File: FiVL/pipeline/augment_dataset/run_main.sh (hunk 1 above)
Comment (gbenms) [reply FiVL/pipeline/augment_dataset/run_main.sh]: done

File: FiVL/pipeline/augment_dataset/utils_preprocess_image.py (hunk 2 above)
Comment (gbenms) [reply FiVL/pipeline/augment_dataset/utils_preprocess_image.py]: done

**Pull request merged**